nilearn/input_data/tests/test_niftispheresmasker.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
```

Comment:
Please add these tests in the existing file nilearn/input_data/tests/test_nifti_spheres_masker.py.